app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -77,7 +77,7 @@ private class SessionsAdapter(val context: Context) : RecyclerView.Adapter<Sessi
         return ViewHolder(textView)
     }
 
-    override fun getItemCount(): Int = 0
+    override fun getItemCount(): Int = 100
```

Comment:
LOL

Reply:
Ahah, I actually didn’t mean to commit that! But it does help demonstrate the motionlayout 😹😹😹